amazon/iontest/ion_test_driver.py

```diff
@@ -55,6 +56,10 @@
                                         `ion-test-driver-results.ion` under the directory specified by the
                                         `--output-dir` option.
 
+    -R, --results-diff <description1>   Given two implementation descriptions of the form name,revision. Name is the
+    <description2>                      implementation's name and revision should be commit hash. Analyze an existing
```

Comment:
I don't think this format will work with docopt. You'll probably need to have `-R, --results-diff <description1> <description2>` all on the same line.